File: src/sdk-reference/java/1/auth/check-token/snippets/check-token.java
try {
    String token = kuzzle.getAuth().login("local", "{\"username\":\"foo\",\"password\":\"bar\"}");
    TokenValidity res = kuzzle.getAuth().checkToken(token);
    if (res.getValid() == true)
      System.out.println("Success");
    else
      System.err.println(res.getState());
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}

File: src/sdk-reference/js/6/auth/check-token/snippets/check-token.js
const credentials = { username: 'foo', password: 'bar' };

try {
  const jwt = await kuzzle.auth.login('local', credentials);
  
  const result = await kuzzle.auth.checkToken(jwt);

  if (result.valid) {
    console.log('Success');
  } else {
    console.error(result.state);
  }
} catch (error) {
  console.error(error.message);
}
